component_tests/rest_api_server/src/repos/UserRepo.ts

```diff
@@ -0,0 +1,88 @@
+import { IUser } from '@src/models/User';
+import { getRandomInt } from '@src/util/misc';

```

Comment:
is all of this code useful for the demo? could perhaps make a more minimal express js example if not e.g. https://github.com/expressjs/express/blob/master/examples/route-map/index.js

Reply:
yeah its a skeleton generated by running `npx express-generator-typescript`. I was kind of thinking the user scaffolding might be useful later if we want to add other test code to this server, especially for testing internet accounts